powerauth-restful-security-javaee/src/main/java/io/getlime/security/powerauth/rest/api/jaxrs/service/v3/TokenService.java

```diff
@@ -99,8 +99,8 @@ public EciesEncryptedResponse createToken(EciesEncryptedRequest request,
             response.setEncryptedData(token.getEncryptedData());
             return response;
         } catch (Exception ex) {
-            logger.warn("Creating PowerAuth token failed", ex);
-            throw new PowerAuthAuthenticationException(ex.getMessage());
+            logger.warn("Creating PowerAuth token failed, error: {}", ex.getMessage());
+            throw new PowerAuthAuthenticationException("POWER_AUTH_TOKEN_ERROR");
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.